index.html

```diff
@@ -0,0 +1,73 @@
+                        <li><img class="logo" src="images/apple.png" alt="apple-icon"></li>
+                        <li><a href="#">Store</a></li>
+                        <li><a href="#">Mac</a></li>
+                        <li><a href="#">iPod</a></li>
+                        <li><a href="#">iPhone</a></li>
+                        <li><a href="#">iPad</a></li>
+                        <li><a href="#">iTunes</a></li>
+                        <li><a href="#">Support</a></li>
+                    </ul>
+                    <form class="nav-form" action="submit">
+                        <img src="images/search.png" alt="search">
+                        <input class="search" type="text">
+                    </form>
+                </nav>
+
+                <div class="titleMain">
+                    <div class="titleHeader"><a href="#"></a></div>
+                    <div class="titleSubHeader"><a href="#"></a></div>
+                    <div class="titleLink"><a href="#"></a></div>
+                </div>
+            </div>
+        </div>
+
+        <div class="middle-section flex-fix">
+            <a href="#" class="first-middle-img middle-section-image"></a>
+            <a href="#" class="second-middle-img middle-section-image"></a>
+            <a href="#" class="third-middle-img middle-section-image"></a>
+            <a href="#" class="fourth-middle-img middle-section-image"></a>
+        </div>
+    </main>
+
+    <footer class="lower-section footer-txt">
+        <div class="bottom-box">
+            <div class="footer-content-1"><p>shop <a href="#" class="footer-link">Apple Online Store</a>(1-800-MY-APPLE), visit an <a href="#" class="footer-link">Apple Retail Store</a>, or find <a href="#" class="footer-link">reseller</a>.</p>
+            </div>
+
+            <div class="footer-content-2">
+                <ul class="footer-menu flex-fix">
+                    <li><a href="#" class="footer-link">Site Map</a></li>
+                    <li><a href="#" class="footer-link footer-lists">Hot News</a></li>
+                    <li><a href="#" class="footer-link footer-lists">RSS Feeds</a></li>
+                    <li><a href="#" class="footer-link footer-lists">Media Info</a></li>
+                    <li><a href="#" class="footer-link footer-lists">Environment</a></li>
+                    <li><a href="#" class="footer-link footer-lists">Job Opportunities</a></li>
+                    <li><a href="#" class="footer-link footer-lists">Contact Us</a></li>
+                    <li><a href="#" class="footer-link footer-lists"><img src="images/us.png" alt=""></a></li>
+                </ul>
+            </div>
+
+            <div class="footer-content-3">
+                <p>copyright &copy; 2014 Apple Inc. All right reserved. <a href="#" class="footer-link">Terms of use</a><a href="#" class="footer-link footer-lists">Privacy policy</a></p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] The layout of this section is really different. Make it look more like the original.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88217523-0bb58580-cc35-11ea-8d95-b6f6a2f38025.png) | ![image](https://user-images.githubusercontent.com/61479648/88217548-153eed80-cc35-11ea-8c3d-b8f9638d38d0.png) |